config/initializers/01_obis_setup.rb

```diff
@@ -82,6 +82,13 @@
   USE_FEEDBACK_LINK                         = application_config['use_feedback_link'] || false
   FEEDBACK_LINK                             = application_config['feedback_link'] || "#"
   NAVBAR_LINKS                              = application_config['navbar_links'] || {}
```

Comment:
@Stuart-Johnson @jleonardw9 @loganfreeman 
Note: These constants will probably need to be added to the OS Application.yml database migration